python/fbprophet/forecaster.py

```diff
@@ -1170,6 +1170,7 @@ def fit(self, df, **kwargs):
         else:
             self.params = self.stan_backend.fit(stan_init, dat, **kwargs)
 
+        self.stan_fit = self.stan_backend.stan_fit
```

Comment:
I added this because `forecaster.stan_fit` is currently `None`. I think this is a clearer way to expose the model object (rather than having to dig into `self.stan_backend`).